Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml

```diff
@@ -64,19 +64,6 @@
   - type: StaticPrice
     price: 750
 
-- type: entity
-  parent: ClothingShoesBootsMag
-  id: ClothingShoesBootsMagBlinding
-  name: magboots of blinding speed
-  description: These would look fetching on a fetcher like you.
-  components:
-  - type: ClothingSpeedModifier
-    walkModifier: 1.10 #PVS isn't too much of an issue when you are blind...
-    sprintModifier: 1.10
-    enabled: false
-  - type: StaticPrice
-    price: 3000
```

Comment:
keep these or else you're going to break maps and downstreams. Eventually these should have the blinding effect added back, which will help them be separate.